NOTICE.txt

```diff
@@ -4680,3 +4680,30 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+---
+
+## Elastic
```

Comment:
Yes, some elastic search PRs were merged in: https://github.com/mattermost/platform/pulls?q=is%3Apr+author%3Agrundleborg+is%3Aclosed

This is the PR that added the dependency https://github.com/mattermost/platform/pull/6340